soft4pes/utils/conversions.py

```diff
@@ -7,17 +7,18 @@
 
 def abc_2_alpha_beta(abc):
     """
-    Convert data from abc-frame to alpha-beta frame.
+    Convert quantity from abc-frame to alpha-beta frame using reduced 
```

Comment:
(Here and in all other instances): Convert a quantity

also: using the reduced